crypto/Cargo.toml

```diff
@@ -8,7 +8,10 @@ edition = "2021"
 
 [features]
 default = ["std"]
+# Use rust standard library
 std = ["ursa"]
+# Force static linking
+vendored = ["openssl-sys"]
```

Comment:
I need this when building for `x86_64-unknown-linux-musl` target which I use to hot-swap Iroha binaries inside the running docker container.

Reply:
Also allowing us to use a smaller container in the future.

Reply:
I would also appreciate if we had a short guide on how you do that. AFAIK most of our users just rebuild every time there's a problem.

Reply:
yes, I'll make a short guide. It's been too tiresome to rebuild image every time

Reply:
#2119